collector/src/test/java/com/google/cloud/trace/zipkin/PlainTextNettyChannelProvider.java

```diff
@@ -0,0 +1,37 @@
+package com.google.cloud.trace.zipkin;
+
+import com.google.auto.service.AutoService;
+import io.grpc.ManagedChannelProvider;
+import io.grpc.netty.NettyChannelBuilder;
+
+/**
+ * Allows non-ssl grpc connections for {@link NettyChannelBuilder} instances.
+ */
+@AutoService(ManagedChannelProvider.class)
+public class PlainTextNettyChannelProvider extends ManagedChannelProvider
+{
+  @Override
+  public boolean isAvailable()
+  {
+    return true;
+  }
+
+  @Override
+  public int priority()
+  {
+    //put it in front of all other providers: NettyChannelProvider/OkHttpChannelProvider
```

Comment:
I'm a little hesitant about this part. Right now this is in a test artifact, but if future refactorings accidentally moved this into a source artifact, consumers may accidentally start making plaintext gRPC calls in their production code.

If it would help, we can change the SDK to add a constructor to the Sink that accepts a TraceServiceBlockingStub instead of just the apiHost and credentials. Then you could override a TraceServiceBlockingStub bean in the test to use the StackdriverMockServer. 

You could also use a mock Sink instead. That would test that the Stackdriver storage component is used, but wouldn't cover quite as much as your current test.

Thoughts?